futures-select-macro/src/lib.rs

```diff
@@ -168,13 +168,25 @@ pub fn select(input: TokenStream) -> TokenStream {
         .zip(variant_names.iter())
         .map(|(expr, variant_name)| {
             match expr {
-                // Don't bind futures that are already a path.
-                // This prevents creating redundant stack space
-                // for them.
-                syn::Expr::Path(path) => path,
+                syn::Expr::Path(path) => {
+                    // Don't bind futures that are already a path.
```

Comment:
Can you also add a comment here explaining that these futures must still implement `Unpin`?

Reply:
Additionally, extending the `select!` doc comment to explain this would be super useful.

Reply:
Done